### DISCUSSION TITLE
Github android app freezes when opening repository

### DISCUSSION THREAD

**Discussion by rh-id** [community/community#67456] [Mobile]
It freezes when opening this https://github.com/yt-dlp/yt-dlp

**Comment by stevepopovich** [comment]
Hi @rh-id thanks for writing in. Can you please share some more info: What version of the app are you using? iOS or Android? What device are you using. Thanks!

**Reply by rh-id** [reply]
Android app : Github Mobile v1.130.0 (744)
Android Version : ColorOS V12.1
Device : OPPO A53

**Reply by stevepopovich** [reply]
Thanks you! We will look into this.
